rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [OPTIONAL] It is adviseable to use shortcut for setters and getters. You can check this [article](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor`) for more info.
- [ ]  Getters and Setters for @book and @person instance variables should be present

```
  attr_accessor :date, :book, :person

```